ansible/idr-searchengine.yml

```diff
@@ -1,4 +1,4 @@
-# Search ngine +  Elasticsearch
+# Configure Search engine +  run Elasticsearch nodes
```

Comment:
As per https://github.com/IDR/deployment/pull/391#issuecomment-1410024671, I understand our goal is to redeploy the search engine service without changing elastic search. To achiev this, would it make sense to have the first playbook focusing solely on elasticsearch and the second on searchengine?
Practically, this would mean migrating all tasks configuring  `{{ apps_folder }}/searchengine/searchengine` to the second playbook.